### PR TITLE
Add scene scraper for Girls Gone Wild

### PR DESCRIPTION
**Pull request by devnsfw**
This has worked so far for all scenes I've thrown at it. The site offers performers URLs, but the information is not worth it (just name, picture and nil birthdate) so I didn't include it.

### CODE REVIEW

**Comment by DogmaDragon** [comment]
Please rename the file to `GirlsGoneWild.yml`.

File: scrapers/GGW.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - girlsgonewild.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[contains(@class,"watch__title")]/text()
+      Details: //div[contains(@class,"text-mob-more")]/text()
```
Comment (DogmaDragon) [inline scrapers/GGW.yml:11]: Change to `Details: //div[contains(@class,"text-mob-more")]`. Currect selector cuts off details after certain point.

File: scrapers/GGW.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      Performers:
+        Name: //span[contains(@class,"watch__featuring_models")]/a
+      Studio:
+        Name: "Girls Gone Wild"
+        URL: "https://girlsgonewild.com"
```
Comment (DogmaDragon) [inline scrapers/GGW.yml:16]: Change to
```yaml
      Studio:
        Name: //div[@class='p-md text-light mb-5 genres-list']/a
        URL: //div[@class='p-md text-light mb-5 genres-list']/a/@href
```

They have multiple studios listed. https://girlsgonewild.com/studios

**Comment by DogmaDragon** [comment]
Tested:
- Scenes work

**Comment by devnsfw** [comment]
I found that this site is managed by Pornworld, so perhaps the URL should be added to that scraper instead.

**Comment by Maista6969** [comment]
> I found that this site is managed by Pornworld, so perhaps the URL should be added to that scraper instead.

Would you like to resolve that in this PR or should we close it in favor of a new one?

**Comment by devnsfw** [comment]
closing this for now

**Pull request closed without merging**